frontend/src/metabase/styled-components/layout/FullWidthContainer.tsx

```diff
@@ -10,12 +10,12 @@ export const FullWidthContainer = styled.div`
   width: 100%;
 
   ${breakpointMinSmall} {
-    padding-left: 2em;
-    padding-right: 2em;
+    padding-left: 1.5rem;
+    padding-right: 1.5rem;
   }
 
   ${breakpointMinMedium} {
-    padding-left: 3em;
-    padding-right: 3em;
+    padding-left: 2rem;
```

Comment:
Are these numbers from the design somewhere? It looks good though.

Maybe it's also worth noting that questions don't seem to have padding in mobile view.
![image](https://user-images.githubusercontent.com/1937582/178222731-f5064202-235c-468a-b45a-dc35d26181b1.png)

Reply:
Ahh, I don't think I meant to adjust `breakpointMinMedium`. I'll double check that.

Questions Mobile view is coming in another PR